fe/fe-core/src/main/java/com/starrocks/leader/ReportHandler.java

```diff
@@ -1216,6 +1225,73 @@ private static void handleSetTabletEnablePersistentIndex(long backendId, Map<Lon
+
+        TabletInvertedIndex invertedIndex = GlobalStateMgr.getCurrentInvertedIndex();
+        for (TTablet backendTablet : backendTablets.values()) {
+            for (TTabletInfo tabletInfo : backendTablet.tablet_infos) {
+                if (!tabletInfo.isSetBinlog_config_version()) {
+                    continue;
+                }
+                long tabletId = tabletInfo.getTablet_id();
+                long beBinlogConfigVersion = tabletInfo.binlog_config_version;
+                TabletMeta tabletMeta = invertedIndex.getTabletMeta(tabletId);
+                long dbId = tabletMeta != null ? tabletMeta.getDbId() : TabletInvertedIndex.NOT_EXIST_VALUE;
+                long tableId = tabletMeta != null ? tabletMeta.getTableId() : TabletInvertedIndex.NOT_EXIST_VALUE;
+
+                Database db = GlobalStateMgr.getCurrentState().getDb(dbId);
+                if (db == null) {
+                    continue;
+                }
+                db.readLock();
+
+                boolean needToCheck = false;
+                OlapTable olapTable = (OlapTable) db.getTable(tableId);
+                try {
+                    if (olapTable == null) {
+                        continue;
+                    }
+
+                    BinlogConfig binlogConfig = olapTable.getCurBinlogConfig();
+                    // backward compatible
+                    if (binlogConfig == null) {
+                        continue;
+                    }
+                    Long feBinlogConfigVersion = binlogConfig.getVersion();
+                    if (beBinlogConfigVersion < feBinlogConfigVersion) {
+                        tabletToBinlogConfig.add(new ImmutableTriple<>(tabletId, tabletInfo.schema_hash,
+                                olapTable.getCurBinlogConfig()));
+                    } else if (beBinlogConfigVersion == feBinlogConfigVersion) {
+                        if (olapTable.enableBinlog() && olapTable.getBinlogAvailableVersion().size() == 0) {
+                            // not to check here is that the function may need to get the write db lock
+                            needToCheck = true;
+                        }
+                    } else {
+                        LOG.warn(String.format("table %s binlog config version of tabletId: %d, BeId: %d, is %d " +
+                                        "greater than version of FE, which is %d", olapTable.getName(), tabletId, backendId,
+                                beBinlogConfigVersion, feBinlogConfigVersion));
+                    }
+                } finally {
+                    db.readUnlock();
+                }
+
+                if (needToCheck) {
+                    GlobalStateMgr.getCurrentState().getBinlogManager().checkAndSetBinlogAvailableVersion(db,
+                            olapTable, tabletId, backendId);
+                }
+            }
+        }
+
+        LOG.info("find [{}] tablets need set binlog config ", tabletToBinlogConfig.size());
```

Comment:
Set log level to debug?

Reply:
I see the log level of other similar methods is info